packages/ui/src/components/atoms/Button/styles.scss

```diff
@@ -100,7 +100,7 @@
 
   // Small
   --fs-button-small-padding                         : var(--fs-spacing-0) var(--fs-spacing-1);
-  --fs-button-small-min-height                      : var(--fs-spacing-5);
+  --fs-button-small-min-height                      : var(--fs-spacing-7);
```

Comment:
Fixes LH SEO.
Double check this button + Icon layout with @renatamottam 
Should we increase the Icon as well?
<img width="484" alt="Screenshot 2023-04-21 at 16 20 49" src="https://user-images.githubusercontent.com/11325562/233717828-dfcd816a-428a-457c-bc83-4d7fd6fad2ed.png">